src/shadowbox/server/server_access_key.ts

```diff
@@ -212,8 +220,11 @@ export class ServerAccessKeyRepository implements AccessKeyRepository {
     for (const accessKey of this.accessKeys) {
       const usageBytes = bytesTransferredById[accessKey.id] || 0;
       const wasOverDataLimit = accessKey.isOverDataLimit;
-      accessKey.isOverDataLimit =
-          this.accessKeyDataLimit ? usageBytes > this.accessKeyDataLimit.bytes : false;
+      let limitBytes = (accessKey.dataLimit || this._defaultDataLimit)?.bytes;
+      if (limitBytes === undefined) {
+        limitBytes = Number.POSITIVE_INFINITY;
+      }
```

Comment:
This is something you can do in TS4:
```suggestion
      limitBytes ??= Number.POSITIVE_INFINITY;
```

See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#short-circuiting-assignment-operators

Reply:
We're on 3.8.3.  I'll update our dependencies (including to TS 4) in a separate PR